src/css/Layout.scss

```diff
@@ -1,12 +1,27 @@
 .layout {
   font-family: "fira", sans-serif;
   font-size: 15px;
-  width: 300px;
   text-shadow: 2px 2px 2px rgba(0, 0, 0, 0.5);
+  position: relative;
   -webkit-touch-callout: none;
   -webkit-user-select: none;
   -khtml-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
```

Comment:
Do we need these? I don't think we support any of these ancient browsers anymore (we should check if any is still needed though)

Reply:
Based on the [compatiblity table](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select), most browsers support the `user-select` properly, but a lot of them require it to be `-webkit-user-select`.

![chrome_fqQJpyqebA](https://user-images.githubusercontent.com/8262173/73588001-c9cb3400-4488-11ea-8972-bb9fd0023213.png)

Based on this, I think we can keep `-webkit-touch-callout` and `-webkit-user-select` but get rid of the rest.